crates/adapters/src/controller/mod.rs

```diff
@@ -2435,37 +2426,60 @@ impl StorageThread {
 
     /// Thread function.
     fn storage_thread(
-        usage: Arc<AtomicI64>,
-        storage_bytes: Arc<AtomicU64>,
-        storage_mb_secs: Arc<AtomicU64>,
+        controller_status: Arc<ControllerStatus>,
```

Comment:
so this is still the storage thread?
will there be a non-storage thread?

Reply:
The name is now StatisticsThread, I don't know why you're saying it's the storage thread.

Reply:
The function name